#Servico para calcular histograma
int32 flag
---
bool isTheSame
